### ISSUE TITLE
Dynamic inventory

### ISSUE THREAD

**Issue by ad-m** [scarolan/ansible-terraform#4]
In order to effectively demonstrate the possibility of integrating Ansible with Terraform, it proposes to choose a cloud service provider that has dynamic inventory for Ansible.

**Comment by scarolan** [comment]
Agree, this would be a great feature.  IBM/RedHat/Hashi may have developed a solution since this was posted 5 years ago.
